src/elements/content-sidebar/README.md

```diff
@@ -37,6 +37,7 @@ var ContentSidebar = require('./ContentSidebar').default;
 | hasActivityFeed | boolean | `false` | *See the [developer docs](https://developer.box.com/docs/box-content-sidebar#section-options).* |
 | hasMetadata | boolean | `false` | *See the [developer docs](https://developer.box.com/docs/box-content-sidebar#section-options).* |
 | hasSkills | boolean | `false` | *See the [developer docs](https://developer.box.com/docs/box-content-sidebar#section-options).* |
+| hasVersions | boolean | `false` | *See the [developer docs](https://developer.box.com/docs/box-content-sidebar#section-detailssidebarprops).* |
```

Comment:
Should we point to the same `section-options` area of the docs? We have access to update the docs once this is fully released.